bin/ChakraCore/ChakraCore.vcxproj

```diff
@@ -148,7 +148,7 @@
     <ClInclude Include="TestHooks.h" />
   </ItemGroup>
   <!-- Conditionally include the group of files to sign because MBv2 knows what to do with this but the old build definition gets confused. -->
-  <ItemGroup Condition="'$(VSO_MICROBUILD_V2)'=='True'">
+  <ItemGroup Condition="'$(VSO_MICROBUILD_V2)'=='True' AND '$(POGO_TYPE)'!='PGI'">
```

Comment:
Real-signing takes 3-5 minutes per binary and (I guess) depending on load on the signing server. We never release the PGI binaries and they can be used to produce PGO counts even unsigned so there's no reason to sign them. Right now we sign ChakraCore.dll and ch.exe twice. This cuts out a fair amount of time when the _pogo build definition runs.

Also, I think they would prefer we don't unnecessarily Real-sign anything we're not planning on actually releasing.